### PR TITLE
[wasm][sdk] Clean the `managed` directory files on `/t:Clean` build target

### PR DESCRIPTION
**Pull request by kjpou1**
Fix Clean target.

### CODE REVIEW

**Comment by kjpou1** [comment]
@monojenkins build failed

File: sdks/wasm/sdk/Mono.WebAssembly.Build/Mono.WebAssembly.Build.targets [hunk 1]
```diff
@@ -81,4 +88,10 @@
 			OutputFile="$(_MonoWasmRuntimeJs)"
 		/>
   </Target>
+	<Target Name = "CleanManaged">
```
Comment (mhutch) [inline sdks/wasm/sdk/Mono.WebAssembly.Build/Mono.WebAssembly.Build.targets]: I would probably give this target a more explicit name and prefix it with an underscore to indicate that users should not call it directly, e.g. `_CleanWasmManagedOutput`

Comment (kjpou1) [reply sdks/wasm/sdk/Mono.WebAssembly.Build/Mono.WebAssembly.Build.targets]: Addressed in recent commit

**Comment by lewing** [comment]
@monjenkins build failed

**Comment by kjpou1** [comment]
@monjenkins build failed

**Comment by kjpou1** [comment]
@monjenkins build failed

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by kjpou1** [comment]
@monojenkins build failed

**Pull request merged**